lnwallet/channel.go

```diff
@@ -1893,6 +1893,33 @@ func (lc *LightningChannel) restorePendingRemoteUpdates(
 				payDesc.LogIndex)
 		}
 
+		var height uint64
+		var heightSet bool
```

Comment:
style nit: typically we do multi-variable definitions like this:
```suggestion
		var (
		        height uint64
		        heightSet bool
		)
```